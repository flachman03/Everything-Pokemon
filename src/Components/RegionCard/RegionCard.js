import React from 'react'
import './RegionCard.scss'

export const RegionCard = ({region}) => {

  return (
    <button className="region-card__button">{region.name}</button>
  )
}
