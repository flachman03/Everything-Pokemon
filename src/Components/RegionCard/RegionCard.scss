.region-card__button {
  height: 40px;
  border-radius: 10px;
  margin: 5px;
  font-size: 24px;
  &:hover {
    background: hsla(254, 0%, 0%, 0.7);
    color: #fff;
    cursor: pointer;
  }
  &:active {
    background: red;
  }
}