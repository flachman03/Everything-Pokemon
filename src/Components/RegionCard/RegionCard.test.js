import React from 'react'
import { shallow } from 'enzyme'
import { RegionCard } from './RegionCard'

describe('RegionCard', () => {
  let wrapper, mockData;
  beforeEach(() => {
    mockData = {name: 'Khoto'}
    wrapper = shallow(<RegionCard region={mockData} />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should have a button named region-card__button', () => {
    expect(wrapper.find('.region-card__button')).toHaveLength(1)
  })
})