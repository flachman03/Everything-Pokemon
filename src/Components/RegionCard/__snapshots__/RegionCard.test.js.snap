// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegionCard should match the snapshot 1`] = `ShallowWrapper {}`;
